lib/msal-browser/src/operatingcontext/BaseOperatingContext.ts

```diff
@@ -24,20 +29,87 @@ export abstract class BaseOperatingContext {
     protected available: boolean;
     protected browserEnvironment: boolean;
 
+    protected static LOGGER_CALLBACKS: Map<number, ILoggerCallback> = new Map([
```

Comment:
Shouldn't this just be one single callback with a switch/case for the logLevels?

Reply:
I think that the map looks cleaner. I am fine with both approaches given that there is no perf difference.

Reply:
Maybe I'm reading this incorrectly but it looks to me like this map approach results in only logging out messages that are an exact match for the log level configured rather than all messages that match the log level and above. Is that accurate?

Reply:
My original idea was to display messages that match the particular log level. I think you concern is valid. It is better to use a callback to display messages for the provided log level and below. Fixed, thank you.